### PR TITLE
Add execute command

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Close https://github.com/neo-project/neo-express/issues/365.

### CODE REVIEW

File: src/neoxp/Commands/ExecuteCommand.cs [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+            }
+            if (inputScript.TryGetBytesFromBase64String(out var b64Bytes) && TryConvertBytesToScript(b64Bytes, out var s2))
+            {
+                return s2;
+            }
+            return null;
+        }
+
+
+        private bool TryConvertBytesToScript(Span<byte> bytes, out Script script)
```
Comment (Liaojinghui) [inline src/neoxp/Commands/ExecuteCommand.cs:147]:
```suggestion
        private static bool TryConvertBytesToScript(Span<byte> bytes, out Script script)
```

File: src/neoxp/Commands/ExecuteCommand.cs [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+                return true;
+            }
+            catch (Exception)
+            {
+                script = null;
+                return false;
+            }
+        }
+
+        private IEnumerable<string> GetInstructionString(Script script)
```
Comment (Liaojinghui) [inline src/neoxp/Commands/ExecuteCommand.cs:161]:
```suggestion
        private static IEnumerable<string> GetInstructionString(Script script)
```

File: src/neoxp/Commands/ExecuteCommand.cs [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            catch (Exception ex)
+            {
+                app.WriteException(ex, showInnerExceptions: true);
+                return 1;
+            }
+        }
+
+
+
+        private Script? LoadFileScript(string fileName)
```
Comment (Liaojinghui) [inline src/neoxp/Commands/ExecuteCommand.cs:111]:
```suggestion
        private static Script? LoadFileScript(string fileName)
```

File: src/neoxp/Commands/ExecuteCommand.cs [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+                else
+                {
+                    var password = chainManager.Chain.ResolvePassword(Account, Password);
+                    await txExec.ContractInvokeAsync(script, Account, password, WitnessScope, AdditionalGas);
+                }
+
+                console.WriteLine("Opcodes:");
+                foreach (var opinfo in GetInstructionString(script))
+                {
+                    console.WriteLine(opinfo);
```
Comment (Liaojinghui) [inline src/neoxp/Commands/ExecuteCommand.cs:98]:
```suggestion
                    console.WriteLine(opInfo);
```

**Review by Liaojinghui — APPROVED** [review]

**Comment by Liaojinghui** [comment]
@Ashuaidehao please check my updates

File: src/neoxp/Commands/ExecuteCommand.cs [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+        readonly ExpressChainManagerFactory chainManagerFactory;
+        readonly TransactionExecutorFactory txExecutorFactory;
+
+        public ExecuteCommand(ExpressChainManagerFactory chainManagerFactory, TransactionExecutorFactory txExecutorFactory)
+        {
+            this.chainManagerFactory = chainManagerFactory;
+            this.txExecutorFactory = txExecutorFactory;
+        }
+
+        [Argument(0, Description = "A neo-vm script(hex,base64) or a script file path")]
```
Comment (cschuchardt88) [inline src/neoxp/Commands/ExecuteCommand.cs:39]: Can we have it read `A neo-vm script (Format: HEX, BASE64, Filename)`

File: src/neoxp/Commands/ExecuteCommand.cs [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+        [Required]
+        internal string InputText { get; set; } = string.Empty;
+
+        [Option(Description = "Account to pay invocation GAS fee")]
+        internal string Account { get; init; } = string.Empty;
+
+        [Option(Description = "Witness Scope to use for transaction signer (Default: CalledByEntry)")]
```
Comment (cschuchardt88) [inline src/neoxp/Commands/ExecuteCommand.cs:46]: `Witness Scope for transaction signer(s) (Allowed: None, CalledByEntry, Global)`

**Comment by Ashuaidehao** [comment]
All update.

**Review by Liaojinghui — APPROVED** [review]

**Review by cschuchardt88 — APPROVED** [review]

**Pull request merged**